### ISSUE TITLE
I do not want to use the steam browser...

### ISSUE THREAD

**Issue by jersten** [ValveSoftware/steam-for-linux#4260]
Noticed there was a video posted on update news to YouTube. I clicked on it and the embedded browser in steam opens up and starts playing ads from YouTube; this was extremely aggravating... So I go into Settings > Web Browser and there is no option to open my systems default browser.

Since this setting seems to be missing completely; it's more or less a feature request. Certain things should open in the systems default browser or the user should have at least the option to open certain pages in the systems default browser. I have multiple reasons for this:
- The steam browser probably does not block trackers.
- The steam browser probably does not block third-party cookies.
- The steam browser probably is not as vetted for security as other browsers such as Chrome/Firefox.
- The stream browser doesn't support addons to enhance browser security. 

Because of all of the above; there needs to be an option to open content using the system browser.

**Comment by not83** [comment]
"there needs to be an option to open content using the system browser"

Sorry but what does this even mean? What bugs can you reproduce

**Comment by Tele42** [comment]
All it means is have a config option to heavily prefer xdg-open over internal options.

**Comment by jersten** [comment]
Yea, basically it's a feature request to add an option to use the systems default browser instead of the built in steam when when viewing certain content (such as clicking on a YT vid).

**Comment by Walkman100** [comment]
Not saying it's a replacement for this option, but you can always right-click -> `Copy link address` then paste it in your browser.

**Comment by Oxalin** [comment]
Are you using big picture mode?

**Comment by jersten** [comment]
Nope

**Comment by gear4s** [comment]
Right-click into Copy link address sometimes doesn't work for me, I usually have to shift-tab, RC=>Copy, paste into any chat window, hilite it, RC=>Copy again, *then* paste it into my browser. (Ubuntu 15.10)

**Comment by stevenroose** [comment]
Also, the `steamwebhelper` process is super unstable. Right now for me it spawns two of those that at each consume 100% of a CPU thread. Researching this, I found multiple past occasions where this happens and then the issue got "fixed". I still have it.

If we can just disable the web stuff in steam, we can avoid this issues.

**Comment by gear4s** [comment]
oh my lord, necro from almost 3 years ago
